local/gugcat/classes/grade_aggregation.php

```diff
@@ -258,4 +264,20 @@ public static function export_aggregation_tool($course, $modules, $students){
         $exportdata = new ArrayObject($array);
         local_gugcat::export_gcat($filename, $columns, $exportdata->getIterator());
     }
+
+    public static function is_resit($module) {
+        global $DB;
+
+        if($taginstances = $DB->get_records('tag_instance', array('itemid'=>$module->id), null, 'tagid')){
+            foreach($taginstances as $taginstance){
+                $tag = $DB->get_field('tag', 'name', array('id'=>$taginstance->tagid));
+
+                if($tag = 'resit'){
```

Comment:
Can you please make this case insensitive? Maybe they'll name it as Resit or RESIT.